pom.xml

```diff
@@ -141,5 +141,12 @@
             <artifactId>structs</artifactId>
             <version>1.7</version>
         </dependency>
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
```

Comment:
Jenkins core already contains a library for working with JSON: https://github.com/kohsuke/Json-lib (Classes are in the net.sf.json package (e.g. net.sf.json.JSONObject)), so I would prefer to reuse that library instead of adding a new dependency.